src/mame/video/pgm2.cpp

```diff
@@ -99,12 +102,17 @@ inline void pgm2_state::skip_sprite_chunk(int &palette_offset, uint32_t maskdata
 		palette_offset-=bits;
 	}
 
-	palette_offset &= m_sprites_colour_mask;
-
+	palette_offset &= m_sprites_colour.mask();
 }
 
-inline void pgm2_state::draw_sprite_line(const rectangle &cliprect, int &mask_offset, int &palette_offset, int x, int realy, int flipx, int reverse, int sizex, int pal, int zoomybit, int zoomx_bits, int xrepeats)
+inline void pgm2_state::draw_sprite_line(bitmap_rgb32 &bitmap, bitmap_ind8 &primap, const rectangle &cliprect, const pen_t *pen, int &mask_offset,
+                                         int &palette_offset, int x, int realy, int flipx, int reverse, int sizex, int pri, int zoomybit, int zoomx_bits, int xrepeats)
 {
+	if ((realy < cliprect.min_y) && (realy > cliprect.max_y))
```

Comment:
Can this condition ever be true?  For `realy` to be both less than the minimum and greater than the maximum (because `&&`), `cliprect` would have to be an invalid rectangle.